server/serializers/subscriptions.go

```diff
@@ -51,21 +57,33 @@ type SubscriptionList struct {
 }
 
 type CreateSubscriptionRequestPayload struct {
-	Organization   string `json:"organization"`
-	Project        string `json:"project"`
-	EventType      string `json:"eventType"`
-	ServiceType    string `json:"serviceType"`
-	ChannelID      string `json:"channelID"`
-	Repository     string `json:"repository"`
-	RepositoryName string `json:"repositoryName"`
-	TargetBranch   string `json:"targetBranch"`
+	Organization                     string `json:"organization"`
+	Project                          string `json:"project"`
+	EventType                        string `json:"eventType"`
+	ServiceType                      string `json:"serviceType"`
+	ChannelID                        string `json:"channelID"`
+	Repository                       string `json:"repository"`
+	RepositoryName                   string `json:"repositoryName"`
+	TargetBranch                     string `json:"targetBranch"`
+	PullRequestCreatedBy             string `json:"pullrequestCreatedBy"`
+	PullRequestReviewersContains     string `json:"pullrequestReviewersContains"`
```

Comment:
```suggestion
	PullRequestCreatedBy             string `json:"pullRequestCreatedBy"`
	PullRequestReviewersContains     string `json:"pullRequestReviewersContains"`
```

Reply:
these cannot be modified as these are the expected key names for AZD